### PR TITLE
resolve issue when connecting

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

File: frontend/src/components/Form.jsx [hunk 1]
```diff
@@ -30,22 +30,26 @@ function Form() {
...
-        let response = err.response.data;
-        if (response.includes("email_UNIQUE")) {
-          setPage(1);
-          setError("email");
-          notify("Email déjà utilisé");
-        } else if (response.includes("pseudo_UNIQUE")) {
-          setPage(1);
-          setError("pseudo");
-          notify("Pseudo déjà utilisé");
+        console.log(err);
```
Comment (LoicBrassart) [inline frontend/src/components/Form.jsx]: NAN ! è_é

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
I iz unhappy !
(ʘ言ʘ╬)

File: frontend/src/components/Form.jsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/Form.jsx]: oups

**Review by LoicBrassart — APPROVED** [review]
No More Evil

**Review by Gumatan — APPROVED** [review]

**Pull request merged**